src/bundle/base58.zig

```diff
@@ -0,0 +1,589 @@
+const std = @import("std");
+
+// Base58 alphabet (no '0', 'O', 'I', or 'l' to deter visual similarity attacks.)
+const base58_alphabet = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
+
+/// We use 256-bit BLAKE3 hashes
+const hash_bytes = 32;
+
+/// Number of Base58 characters needed to represent a 256-bit hash
+///
+/// Math:
+/// - Each base58 character can represent 58 values
+/// - So we need ceil(log58(MAX_HASH + 1)) characters
+/// - Which is ceil(log(MAX_HASH + 1) / log(58))
+/// - Since MAX_HASH is 2^256 - 1, we need ceil(256 * log(2) / log(58))
+/// - log(2) / log(58) ≈ 0.17329
```

Comment:
I think this calculation is wrong: https://www.wolframalpha.com/input?i=log_58%282%5E256%29